pkg/ipcache/kvstore.go

```diff
@@ -97,7 +98,7 @@ func newKVReferenceCounter(s store) *kvReferenceCounter {
 
 // UpsertIPToKVStore updates / inserts the provided IP->Identity mapping into the
 // kvstore, which will subsequently trigger an event in NewIPIdentityWatcher().
-func UpsertIPToKVStore(ctx context.Context, IP, hostIP net.IP, ID identity.NumericIdentity, key uint8,
+func UpsertIPToKVStore(ctx context.Context, IP, hostIP *netip.Addr, ID identity.NumericIdentity, key uint8,
```

Comment:
Why does this still need to be a pointer?

Reply:
It doesn't. Thanks for catching this.